### ISSUE TITLE
Problem with switching working directory

### ISSUE THREAD

**Issue by pmkowal** [tadija/bitrise-step-xcode-project-info#1]
Hi, I've noticed that when the info plist is placed in the path that contains white signs, the path is truncated.

```
[INFO] Switching working dir to [Project/Supporting Files]
line 24: cd: Project/Supporting: No such file or directory
```

I think adding double quotes in the line 24 would do the job:
`cd "$DIRECTORY"` instead of `cd $DIRECTORY`

**Comment by tadija** [comment]
Hi, thanks for reporting this! 
It is fixed now in version 1.0.2, I've just made a pull request to merge it in bitrise-steplib.

**Comment by pmkowal** [comment]
Great, thanks! 👍
